fizzbuzz.rb

```diff
@@ -0,0 +1,13 @@
+num = 1
+while num <= 20 
+    if num % 3 == 0 && num % 5 == 0
+        puts "FizzBuzz"
+    elsif num % 3 == 0
+        puts "Fizz"
+    elsif num % 5 == 0
+        puts "Buzz"
+    else
+        puts "#{num}"
+    end
+    num = num + 1
```

Comment:
自己代入演算子というものを使うと少し簡略化できたりします〜
https://docs.ruby-lang.org/ja/latest/doc/spec=2foperator.html#selfassign
```suggestion
    num += 1
```